packages/backend/server/src/models/copilot-workspace.ts

```diff
@@ -34,6 +34,48 @@ export class CopilotWorkspaceConfigModel extends BaseModel {
     });
   }
 
+  @Transactional()
+  private async listEmbeddedDocIds(
+    workspaceId: string,
+    options?: PaginationInput
+  ) {
+    return await this.db.aiWorkspaceEmbedding.findMany({
+      where: {
+        workspaceId,
+      },
+      select: {
+        docId: true,
+        createdAt: true,
+      },
+      orderBy: { createdAt: 'desc' },
+      skip: options?.offset,
+      take: options?.first,
+    });
+  }
+
+  /**
+   * find docs to embed, excluding ignored and already embedded docs
+   * newer docs will be list first
+   * @param workspaceId id of the workspace
+   * @returns docIds
+   */
+  @Transactional()
+  async findDocsToEmbed(workspaceId: string): Promise<string[]> {
+    const docIds = await this.db.snapshot
+      .findMany({
+        where: { workspaceId },
+        select: { id: true },
+      })
+      .then(r => r.map(doc => doc.id));
+
+    const skipDocIds = await Promise.all([
+      this.models.copilotWorkspace.listEmbeddedDocIds(workspaceId),
```

Comment:
try 'none' https://www.prisma.io/docs/orm/reference/prisma-client-reference#none

or at least `this.listEmbededDocIds`

Reply:
done